tardis/montecarlo/tests/test_packet_source.py

```diff
@@ -43,27 +52,41 @@ def blackbodysimplesource(self, request):
         tardis.montecarlo.packet_source.BlackBodySimpleSource
         """
         cls = type(self)
+        _ = config
         montecarlo_configuration.LEGACY_MODE_ENABLED = True
         cls.bb = BlackBodySimpleSource(base_seed=1963, legacy_second_seed=2508)
+        self.regression_data.sync_hdf_store(cls.bb, update_fname=False)
         yield cls.bb
         montecarlo_configuration.LEGACY_MODE_ENABLED = False
 
     @pytest.fixture(scope="class")
-    def blackbody_simplesource_relativistic(self, request):
+    def blackbody_simplesource_relativistic(self, request, config):
         """
         Create BlackBodySimpleSourceRelativistic instance.
 
         Yields
         -------
         tardis.montecarlo.packet_source.BlackBodySimpleSourceRelativistic
         """
+        _ = config
         montecarlo_configuration.LEGACY_MODE_ENABLED = True
         bb_rel = BlackBodySimpleSourceRelativistic(
             base_seed=1963, legacy_second_seed=2508
         )
+        self.regression_data.sync_hdf_store(bb_rel, update_fname=False)
         yield bb_rel
         montecarlo_configuration.LEGACY_MODE_ENABLED = False
 
+    # def test_blackbody_regressions(
+    #     self, blackbodysimplesource
+    # ):
+    #     expected = pd.read_hdf(
+    #         self.regression_data.fpath, key="/black_body_simple_source/scalars"
+    #     )
+    #     np.testing.assert_allclose(
+    #         expected.base_seed, blackbodysimplesource.base_seed
+    #     )
```

Comment:
@andrewfullard Do we want to keep this test? Or do I need to change some other test module to test this change?

Reply:
If we could test this for the entire source output, that would be very useful.

Reply:
the packet source at this moment only has the base seed, other attributes are none I'm afraid